web/src/testgrid-grid-row.ts

```diff
@@ -0,0 +1,41 @@
+import { LitElement, html, css } from "lit";
+import { map } from "lit/directives/map.js";
+import { customElement, property } from "lit/decorators.js";
+import { ListRowsResponse_Row } from './gen/pb/api/v1/data.js';
+import { TestStatus } from '../src/gen/pb/test_status/test_status';
+
+interface Cell {
```

Comment:
would it be better to define this interface in the cell component and import it here?

Reply:
Actually just deleted it, since it's not used currently.